.travis.yml

```diff
@@ -18,3 +18,9 @@ deploy:
     node: '4'
     tags: true
     repo: thelounge/lounge
+
+script:
```

Comment:
I wonder, perhaps we should be invoking build commands too? They could potentially fail.